src/test/shared/credentials/credentialsProfileMru.test.ts

```diff
@@ -7,24 +7,11 @@
 
 import * as assert from 'assert'
 import { CredentialsProfileMru } from '../../../shared/credentials/credentialsProfileMru'
-import { SettingsConfiguration } from '../../../shared/settingsConfiguration'
+import { TestSettingsConfiguration } from '../../utilities/testSettingsConfiguration'
 
-class TestSettingsConfiguration implements SettingsConfiguration {
+describe('CredentialsProfileMru', () => {
 
-    private readonly _data: { [key: string]: any } = {}
-
-    public readSetting<T>(settingKey: string, defaultValue?: T | undefined): T | undefined {
-        return this._data[settingKey] as T
-    }
-
-    public async writeSetting<T>(settingKey: string, value: T, target: any): Promise<void> {
-        this._data[settingKey] = value
-    }
-}
-
-describe('CredentialsProfileMru', function(): void {
-
-    it('lists single profile when only one exists', async function() {
+    it('lists single profile when only one exists', async () => {
```

Comment:
Nit: While you're here, can you add a test for the case where no profiles exist?